### PR TITLE
SHOT22-163 Merge localization

### PR DESCRIPTION
**Pull request by willis102**
- Merges remaining localizations per SHOT22-163
- Applies Shotgun -> ShotGrid rebranding to localizations
- Fixes syntax error in translation that was breaking `ko` build

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/2331b7200b9211899e5f3a9d74bde191ac1aa6cb/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/66b426c8643bcf4f56d4625099763c679e9acdbc/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/8cf1401d82d859860691ac5735410d1ae4e9b773/index.html)

File: docs/ko/guides/pipeline-integrations/administration/file-system-config-reference.md [hunk 1]
```diff
@@ -1051,7 +1051,7 @@ UTC 시간으로 기본 설정되는 YYYY.MM.DD 형식의 타임스탬프입니
 
 시퀀스, 샷, 단계 및 버전은 동일한 템플릿 파일에 정의된 키입니다.
 
-{% include info title="참고" content="문자열 키의 이름이 연관된 ShotGrid 엔티티가 있는 동적 스키마 폴더의 엔티티 유형과 일치하면 해당 폴더 이름이 토큰으로 대체됩니다. 예를 들어 위의 코드 조각과 같이 'string' 유형의 {Sequence} 템플릿 키를 사용하고 스키마에서 'sequence'라는 동적 폴더가 있는 경우 해당 `sequence.yml` 파일에서 `shotgun_entity` 유형으로 정의되며 ShotGrid의 "Sequence" 엔티티 유형에 연결됩니다. 툴킷은 템플릿 키가 이 동적 폴더의 엔티티 유형이라는 것을 인식합니다(여기서는 둘 다 Sequence임). 따라서 툴킷은 폴더 이름(즉, 특정 시퀀스의 이름)을 가져와서 템플릿 키로 대체합니다." %}
+{% include info title="참고" content="문자열 키의 이름이 연관된 ShotGrid 엔티티가 있는 동적 스키마 폴더의 엔티티 유형과 일치하면 해당 폴더 이름이 토큰으로 대체됩니다. 예를 들어 위의 코드 조각과 같이 'string' 유형의 {Sequence} 템플릿 키를 사용하고 스키마에서 'sequence'라는 동적 폴더가 있는 경우 해당 `sequence.yml` 파일에서 `shotgun_entity` 유형으로 정의되며 ShotGrid의 'Sequence' 엔티티 유형에 연결됩니다. 툴킷은 템플릿 키가 이 동적 폴더의 엔티티 유형이라는 것을 인식합니다(여기서는 둘 다 Sequence임). 따라서 툴킷은 폴더 이름(즉, 특정 시퀀스의 이름)을 가져와서 템플릿 키로 대체합니다." %}
```
Comment (willis102) [inline docs/ko/guides/pipeline-integrations/administration/file-system-config-reference.md:1054]: This is the fix for the broken build for the `ko` site -- the double quotes were added in translation and cause a syntax error.

Comment (guillaume-brossard-adsk) [reply docs/ko/guides/pipeline-integrations/administration/file-system-config-reference.md:1054]: Looks good.

**Review by guillaume-brossard-adsk — APPROVED** [review]

**Pull request merged**